LetsDinnerV2 MessagesExtension/Controller/NewEventVC/NewEventViewController.swift

```diff
@@ -166,7 +166,10 @@ class NewEventViewController: UIViewController  {
     }
     
      private func allFieldsAreFilled() -> Bool {
-        guard let host = hostNameTextField.text, let dinner = dinnerNameTextField.text, let location = locationTextField.text, let date = dateTextField.text else {
+        guard let host = hostNameTextField.text,
```

Comment:
Does this prevent an empty string to be validated?

Reply:
its used for safely unwrapped the optional

Reply:
The code below is for the validation